package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "frontend",
```

Using the same convention as in the [backend `package.json`](https://github.com/101-Coconsulting/TC3005B.501-Backend/blob/development/package.json) file, this should be renamed to `tc3005b-501-frontend` as well.
```suggestion
  "name": "tc3005b-501-frontend",
```